src/components/Card.js

```diff
@@ -35,31 +33,24 @@ export default class Card {
   isLiked() {
     return this._likes.some((like) => like._id === this._myId);
   }
-  setLikes(likes) {
```

Comment:
There is no need to remove `setLikes` method, you should have it, it should update likes array and render likes. Call it in index.js after updating likes

```js
setLikes(likes) {
  this._likes = likes;
  this.updateLikes();
}
```